helpers/notification.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from enum import Enum
+
+from dag_datalake_sirene.helpers import tchap
+
+
+class Notification:
+    """
+    Class to manage and send end of DAG notifications to Tchap.
+
+    Methods:
+        send_notification_tchap() -> None:
+            Sends a notification to Tchap with the following format:
+                🟠 dagA: Données
+                - task1(success): N rows were updated.
+                - task2(failed)
+
+    Usage:
+        Add the following parameters to a DAG definition:
+            >> on_failure_callback=Notification.send_notification_tchap,
+            >> on_success_callback=Notification.send_notification_tchap,
+
+        [Optional] In the relevant @task, use the following code to provide additional
+        context for the notification:
+            >> from dag_datalake_sirene.helpers import Notification
+            >> ti.xcom_push(key=Notification.notification_xcom_key, value=error_message)
+    """
```

Comment:
I typically use NumPy/SciPy style Docstring for this project, but we can switch to Google style during refactoring if you prefer. 

Though I do admit I must have slipped one a few ones 😅 

The key is to maintain consistency by sticking to one style throughout in my opinion. So we should decide on one.
What do you think ?

Reply:
I wrote it like this out of habits but indeed let's stay consistent. I'll convert it to Numpy style, good point!

Reply:
Should we keep it and merge to Google for all Docstring in the refactor since it's more concise ?

Reply:
After discussion, we'll go the Google route !!